src/data/mock/trbRequest.ts

```diff
@@ -613,12 +616,14 @@ export const trbLeadOptions: MockUserInfo[] = [
   users.next()?.userInfo!
 ];
 
-export const getTrbLeadOptionsQuery: MockedQuery<GetTrbLeadOptions> = {
+export const getTrbLeadOptionsQuery: MockedQuery<GetTrbLeadOptionsQuery> = {
   request: {
-    query: GetTrbLeadOptionsQuery,
+    query: GetTrbLeadOptionsDocument,
     variables: {}
   },
   result: {
+    // readonly __typename is not recognized by ts
```

Comment:
Any reason why this is needed? Seems odd since the types all seem to define `__typename` in the generated file 🤔 

Will we need to put this in all mocks?

Reply:
The mock data here is defining the `__typename` as `readonly __typename`.  Both are referring to the same type.  Let me pull up the github issue where TS has trouble with readonly type matching.  If that's the case

Reply:
Hmm I just tried the same thing I did earlier and ts isn't throwing error.  Earlier was still throwing TS error, maybe a ide refresh?  This should address it

```
data: {
      __typename: 'Query',
      trbLeadOptions
    }
```